File: tfh-backend/src/main/java/com/example/tfhbackend/dto/OrderDTO.java
package com.example.tfhbackend.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Builder
@Data
public class OrderDTO {
    private Long id;
    private Long bookingId;
    private List<MenuItemWithCountDTO> items;

}

File: tfh-mobile/app/src/main/java/com/example/tfhmobile/dto/OrderDTO.kt
package com.example.tfhmobile.dto

class OrderDTO constructor(
    private val id: Long,
    private val bookingId: Long,
    private val items: List<MenuItemWithCountDTO>
) {

    fun getId(): Long {
        return id
    }

    fun getItems(): List<MenuItemWithCountDTO> {
        return items
    }

    fun getBookingId(): Long = bookingId;
}